osu.Framework.Font/Graphics/Containers/MaskingContainer.cs

```diff
@@ -74,7 +74,7 @@ private static RectangleI generateMasking(RectangleI source, Edges edges)
             int getExtendSize(Edges flag)
             {
                 const int extend_size = 1000;
-                return edges.HasFlag(flag) ? 0 : extend_size;
+                return (edges & flag) == flag ? 0 : extend_size;
```

Comment:
Not very sure show use `HasFlagFast` follow some part in the osu-framework?

Reply:
I couldn't figure out how to use HasFlagFast from the error message, I tried just doing edges.HasFlagFast(flag), but that didn't exist on the enum, so I just manually done bit operations here since it didn't seem to exist.

Reply:
HasFlagFast is a extension method in the osu-framework